pom.xml

```diff
@@ -292,6 +288,16 @@
         <artifactId>postgresql</artifactId>
         <version>${postgres.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.h2database</groupId>
+        <artifactId>h2</artifactId>
+        <version>1.4.187</version>
```

Comment:
It's only used once because we can't do `DROP DATABASE` and `CREATE DATABASE ...` in h2.